src/goto-programs/builtin_functions.cpp

```diff
@@ -696,7 +696,10 @@ void goto_convertt::do_havoc_slice(
   // char nondet_contents[argument[1]];
   // __CPROVER_array_replace(p, nondet_contents);
 
-  r_or_w_ok_exprt ok_expr(ID_w_ok, arguments[0], arguments[1]);
+  symbol_exprt deallocated =
+    ns.lookup(CPROVER_PREFIX "deallocated").symbol_expr();
+  symbol_exprt dead = ns.lookup(CPROVER_PREFIX "dead_object").symbol_expr();
+  w_ok_exprt ok_expr{arguments[0], arguments[1], deallocated, dead};
```

Comment:
Do you still need this change?

Reply:
It won’t be subject to cleaning at this point, but I could of course explicitly invoke cleaning.